src/domain/event/EventUtils.ts

```diff
@@ -189,6 +190,15 @@ const getEventLocationFields = (
   };
 };
 
+const getCourseFields = (event: EventFields) => {
```

Comment:
Why is this under EventUtils.ts?

Reply:
hmm, maybe because it is used here in the getEventsFields. Not sure what is the most logical place for this